go.mod

```diff
@@ -1,11 +1,11 @@
 module github.com/bluesky-social/jetstream
 
-go 1.22
+go 1.24
 
-toolchain go1.22.3
+toolchain go1.24.6
```

Comment:
as a small detail, our norm is to set this to `go1.24.1`, not most recent

Reply:
ha i was wondering (`go get` did this, i'm a fake golanger)